Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM $BASE_IMAGE
 # Framework files
 COPY framework/cos/ /
 COPY framework/files/ /
+COPY --from=rancher/system-agent:v0.2.7 /usr/bin/rancher-system-agent /usr/sbin/rancher-system-agent
+COPY --from=ghcr.io/rancher/elemental-operator:dev /usr/bin/elemental-operator /usr/sbin/elemental-operator
```

Comment:
Is elemental using DockerHub or GHCR?

Reply:
Image can't be pulled `Error response from daemon: Head "https://ghcr.io/v2/rancher/elemental-operator/manifests/dev": denied`

Reply:
nobody is pushing to that repo so not sure why this is here :D

Reply:
In the first version it was a privarte repo, it was than changed to this... but as you said, nobody is pushing there.